edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.IoTHub.Test/EdgeAgentConnectionTest.cs

```diff
@@ -2001,7 +2001,7 @@ static object GetEdgeAgentBaseConfiguration()
         {
             var desiredProperties = new
             {
-                schemaVersion = "1.0",
+                schemaVersion = "1.1.0",
```

Comment:
Do we need to concern for the following tests as well? 
- https://github.com/Azure/iotedge/blob/master/edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.IoTHub.Test/reporters/IoTHubReporterTest.cs#L1433
-

Reply:
Those tests seem to be just verifying that the roundtrip payload comes back as expected, and not doing any actual schema verification.